src/Symfony/Component/DependencyInjection/ParameterBag/ParameterBag.php

```diff
@@ -264,6 +219,9 @@ public function escapeValue($value)
         return $value;
     }
 
+    /**
+     * {@inheritdoc}
+     */
     public function unescapeValue($value)
```

Comment:
``resolveValue`` can return different values depending if the ``ParameterBag`` is resolved or not...

See this example:
```php
$parameterBag = new ParameterBag(array(
    'foo' => '100 %%',
));

$parameterBag->resolveValue('%foo% %%'); // will return '100 %% %%'

$parameterBag->resolve();
$parameterBag->resolveValue('%foo% %%'); // will return '100% %%'
```